### ISSUE TITLE
Make special colorization for Entries with References configurable

### ISSUE THREAD

**Issue by gaensler** [keepassxreboot/keepassxc#4880]
## Summary
I propose to make the special colorization for Entries with References configurable via Application Settings. It should be possible to make Entries with References don't appear different from Entries without References.

## Examples
Add an option with toggle button to Application Settings > General > Basic Settings > Entry Management ... i didn't come up with a good name for that option yet :laughing: 
True = Entries with References are greyed out (current behavior and therefore default)
False = Entries with References don't appear different from / have same font color as Entries without References

![image](https://user-images.githubusercontent.com/60502460/85112511-01f3c900-b216-11ea-8354-5d1e678c0f42.png)

I especially don't think there has to be an option to choose the font color, because i read from #3201 and #860 that KeePassXC uses a generic way to detect the desktop theme and colorization of entries with references.

## Context
I work a lot with references between Entries. 
For example i have one entry holding the IP of a Server as URL and several more using that IP to access the server from URL-Field (Double Click)  via different protocols (ssh, https, sql, ...).

![image](https://user-images.githubusercontent.com/60502460/85110568-64979580-b213-11ea-9205-cbf1ce95f6ad.png)

Another example is a group with references to several more KeePass-Files (one for each customer, shared with colleagues via version control). I store the absolut file-path to the directory in one entry and use this referene to only have relative paths in each entry. With this i can easily move the directory with only changing one entry.

In my case the entries i use regularly are greyed out and are not as easy to read and find as the once i only change every x months. Therefore most entries are greyed out and i have only few entries left with "normal text color". This makes the whole entry-list hard to read, because of the little contrast between Text of Entries with references and Background.
In my special case the font color should be vice versa, but i guess that fits for to less users.

I think not everyone or even a little group of users has the need to specially mark Entries with references by different font color, because it's only relevant while editting and not on daily use. 
I can also easily identify Entries with references if they have fields with "Ref:"-Prefix in the Grid and of course with the reference-token in the single entry edit/view.

**Comment by droidmonkey** [comment]
Good point, now with badges in 2.6.0 we could simply make a reference "badge" like a curved arrow that overlays the entry icon instead of changing colors

**Comment by rosaLux161** [comment]
Are there any updates on this?
